README.md

```diff
@@ -1 +1,42 @@
-# Module-1---Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+> This project was done using Html and CSS
+
+
+## Done With
+ 
+ - HTML
+ - CSS
+ - linters
+
+## Authors
+
+👤 *Author1*
+
+- GitHub: [@githubhandle](https://github.com/Rudy-Menekam)
+- Twitter: [@githubhandle](https://twitter.com/MenekamR)
+- LinkedIn: [@githubhandle](https://www.linkedin.com/in/menekam-rudy/)
+
+<!-- - GitHub: [@githubhandle](https://github.com/githubhandle)-->
```

Comment:
- [ ] In my opinion, In order to respect the best practice, we should not keep commented out lines of code. You can learn more about it [here](https://questions.microverse.org/t/is-writing-down-comments-on-the-code-a-good-practice/687).